src/Ecommerce.js

```diff
@@ -0,0 +1,161 @@
+import { useEffect, useState } from 'react';
+import React from 'react-router-dom';
+import axios from 'axios';
+import './Ecommerce.css';
+import Pagination from './Pagination';
+
+const Ecommerce = (props) => {
+
+    const [products, setProducts] = useState();
+    const [currPage, setCurrPage] = useState(1);
+    const [productsPerPage, setProductsPerPage] = useState(9);
+    const [brandNames, setBrandNames] = useState([]);
+    const [filteredProducts, setFilteredProducts] = useState();
+    const [brands, setBrands] = useState();
+    const [sort, setSort] = useState();
+    const [totalProducts, setTotalProducts] = useState();
+
+    useEffect(() => {
+        axios.get('http://localhost:8080/products')
```

Comment:
How would the user know if the get request loading?
in this case you can generate reactLoading component (read in google about react-loading library)